src/acl/AclManager.cc

```diff
@@ -195,11 +196,11 @@ bool AclManager::authorize(
     long long user_req;
     long long resource_oid_req;
 
-    if (static_cast<long long int>(op) & 0x10LL) //No lockable object
+    if (op & 0x10LL) //No lockable object
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We observed that you are using unintended operations or assignments that could lead to logical errors, vulnerabilities, or undefined behavior. To prevent these issues, ensure that you follow best practices, such as using explicit comparisons and clear operator usage, and using parentheses to clarify intent. Learn more - https://cvefeed.io/cwe/detail/cwe-480-use-of-incorrect-operator